src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -228,6 +233,7 @@ public void writeLogTo(XMLOutput out) throws IOException {
 
     @Extension
     public static class DescriptorImpl extends TriggerDescriptor {
+        private static final Logger LOGGER = Logger.getLogger(DescriptorImpl.class.getName());
```

Comment:
I would recommend a logger for <code>GitHubPushTrigger.class</code>. Such approach would be much more useful for logs filtering